Documentation/RenderingDocs/Index.rst

```diff
@@ -10,97 +10,116 @@
+        ..  code-block:: bash
 
-            # verify it worked
-            docker run --rm t3docs/render-documentation --version
+            mkdir -p Documentation-GENERATED-temp
+            docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation
+            xdg-open "Documentation-GENERATED-temp/Index.html"
 
-    #.  Make 'dockrun_t3rd' available in current terminal
+    .. group-tab:: MacOS
 
-        .. code-block:: bash
+        ..  code-block:: bash
 
-            eval "$(docker run --rm t3docs/render-documentation show-shell-commands)"
+            mkdir -p Documentation-GENERATED-temp
+            docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation
+            open "Documentation-GENERATED-temp/Index.html"
 
-        This will run the Docker container and print out shell code. Once "eval-uated" 
-        the code defines a helper function named `dockrun_t3rd`. This function relieves 
-        you of the work of setting the volumes and rights correctly when running the 
-        container and ensures a folder for the rendering results is provided. Executing 
-        the code with `eval` defines the function for your current terminal. Don't forget
-        the quotes around everything that goes into eval. Either define the function each
-        time you open a new terminal window or add the line to the startup file of your
-        shell like `~/.bashrc` or `.zshrc`.
+    ..  group-tab:: Windows
 
-    #.  Run dockrun_t3rd
+        ..  code-block:: powershell
 
-        .. code-block:: bash
+            start "Documentation-GENERATED-temp/Index.html"
+            docker run --rm --pull always -v ${PWD}:/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation
+            open "Documentation-GENERATED-temp/Index.html"
 
-            dockrun_t3rd makehtml
+Rendering extension documentation to docs.typo3.org
+===================================================
 
-        This will automatically find the documentation in the
-        :file:`Documentation` subfolder. It will create a directory
-        :file:`Documentation-GENERATED-temp` and write the results there.
+For your documentation to be rendered to our server https://docs.typo3.org, your
+TYPO3 extension has to have a valid :file:`composer.json` and either a folder
+called :file:`Documentation` with a :file:`Documentation/Index.rst` and
+a :file:`Documentation/guides.xml` or a :file:`README.rst` / :file:`README.md` on
+the level of your :file:`composer.json`.
 
-    #.  Open generated documentation
+The extension has to be publicly available on GitHub or GitLab. You have to
+introduce a :ref:`Webhook <webhook>` and the Documentation Team has to
+:ref:`approve <approval-intercept>` your first rendering request on a tool
+called "Intercept".
```

Comment:
Is Intercept relevant for the user? Can they do something with that knowledge? Maybe link to the docs rendering list, then it has a value.

Reply:
They can have a look at the public part of intercept. It is open for everyone

Reply:
Ok, I removed it, we probably dont have to confuse ppl here